lib/outputs/models.py

```diff
@@ -0,0 +1,170 @@
+        self.RESULT_EXTENDED = report.result_extended
+        self.ACCOUNT_ID = account
+        self.PROFILE = profile
+        self.PROVIDER = report.check_metadata.Provider
+        self.CHECKID = report.check_metadata.CheckID
+        self.CHECKNAME = report.check_metadata.CheckID
+        self.CHECKTITLE = report.check_metadata.CheckTitle
+        self.CHECKTYPE = report.check_metadata.CheckType
+        self.SERVICENAME = report.check_metadata.ServiceName
+        self.SUBSERVICENAME = report.check_metadata.SubServiceName
+        self.RESOURCEIDTEMPLATE = report.check_metadata.ResourceIdTemplate
+        self.SEVERITY = report.check_metadata.Severity
+        self.RESPOURCETYPE = report.check_metadata.ResourceType
+        self.DESCRIPTION = report.check_metadata.Description
+        self.RISK = report.check_metadata.Risk
+        self.RELATED_URL = report.check_metadata.RelatedUrl
+        self.REMEDIATION_RECOMMENDATION_TEXT = report.check_metadata.Remediation[
+            "Recommendation"
+        ]["Text"]
+        self.REMEDIATION_RECOMMENDATION_URL = report.check_metadata.Remediation[
+            "Recommendation"
+        ]["Url"]
+        self.REMEDIATION_RECOMMENDATION_CODE_NATIVEIAC = (
+            report.check_metadata.Remediation["Code"]["NativeIaC"]
+        )
+        self.REMEDIATION_RECOMMENDATION_CODE_TERRAFORM = (
+            report.check_metadata.Remediation["Code"]["Terraform"]
+        )
+        self.REMEDIATION_RECOMMENDATION_CODE_CLI = report.check_metadata.Remediation[
+            "Code"
+        ]["cli"]
+        self.REMEDIATION_RECOMMENDATION_CODE_OTHER = report.check_metadata.Remediation[
+            "Code"
+        ]["other"]
+        self.CATEGORIES = self.__unroll_list__(
+            report.check_metadata.Categories
+        )  # report.check_metadata.Categories
+        self.TAGS = self.__unroll_dict__(report.check_metadata.Tags)
+        self.DEPENDS_ON = self.__unroll_list__(report.check_metadata.DependsOn)
+        self.RELATED_TO = self.__unroll_list__(report.check_metadata.RelatedTo)
+        self.NOTES = report.check_metadata.Notes
+        self.COMPLIANCE = self.__unroll_compliance__(report.check_metadata.Compliance)
+        self.ASSESSMENT_TIME = timestamp.isoformat()
+        self.ACCOUNT_DETAILS_EMAIL = organizations.account_details_email
+        self.ACCOUNT_DETAILS_NAME = organizations.account_details_name
+        self.ACCOUNT_DETAILS_ARN = organizations.account_details_arn
+        self.ACCOUNT_DETAILS_ORG = organizations.account_details_org
+        self.ACCOUNT_DETAILS_TAGS = organizations.account_details_tags
+
+    def __unroll_list__(self, listed_items: list):
+        unrolled_items = ""
+        separator = "|"
+        for item in listed_items:
+            if not unrolled_items:
+                unrolled_items = f"{item}"
+            else:
+                unrolled_items = f"{unrolled_items}{separator}{item}"
+
+        return unrolled_items
+
```

Comment:
What is the difference to unroll a list and unroll compliance list? I think a generic version to unroll values would be great!

Reply:
Since for compliance is neeeded to make a type cast for `Compliance_Framework` the code would look like this:
```
if item is compliance:
#All logic of __unroll_compliance__
else:
# All logic of __unroll_list__
```
I think is better splitted

Reply:
Perfect!